src/patternfly/components/Progress/progress.scss

```diff
@@ -13,13 +13,16 @@
   --pf-c-progress__indicator--BackgroundColor: var(--pf-c-progress__bar--before--BackgroundColor);
 
   // Modifier variables
-  --pf-c-progress--m-danger__bar--BackgroundColor: var(--pf-global--danger-color--100);
   --pf-c-progress--m-success__bar--BackgroundColor: var(--pf-global--success-color--100);
-  --pf-c-progress--m-danger__status-icon--Color: var(--pf-global--danger-color--100);
+  --pf-c-progress--m-warning__bar--BackgroundColor: var(--pf-global--warning-color--100);
+  --pf-c-progress--m-danger__bar--BackgroundColor: var(--pf-global--danger-color--100);
   --pf-c-progress--m-success__status-icon--Color: var(--pf-global--success-color--100);
+  --pf-c-progress--m-warning__status-icon--Color: var(--pf-global--warning-color--100);
+  --pf-c-progress--m-danger__status-icon--Color: var(--pf-global--danger-color--100);
   --pf-c-progress--m-inside__indicator--MinWidth: var(--pf-global--spacer--xl);
   --pf-c-progress--m-inside__measure--Color: var(--pf-global--Color--light-100); // the measure needs to be white when moved inside the progress bar indicator
   --pf-c-progress--m-success--m-inside__measure--Color: var(--pf-global--Color--dark-100); // the measure still needs to be dark when in success state
+  --pf-c-progress--m-warning--m-inside__measure--Color: var(--pf-global--Color--dark-100); // the measure still needs to be dark when in warning state
```

Comment:
you were going to remove this variable `--pf-c-progress--m-success--m-inside__measure--Color` in #3436 so im not sure if you want to add this variable for `pf-m-warning` ?

Reply:
I removed `--pf-c-progress--m-success--m-inside__measure--Color` because the black text on the green background isn't accessible, though I ended up adding it back and setting the text to white per your comment.

Though the warning state text needs to be black against the yellow/orange background.